lms/templates/courseware/course_about.html

```diff
@@ -123,7 +123,7 @@ <h1>
           <span class="register disabled">
             ${_("Course is full")}
           </span>
-        % elif invitation_only and not can_enroll:
+        % elif settings.FEATURES.get('COURSE_DEFAULT_INVITE_ONLY') or invitation_only and not can_enroll:
```

Comment:
Whenever you're doing this sort of thing, please use parens to indicate order of operations. Or better yet, do this calculation in the calling view method and pass in the result as a simple boolean flag to keep the template simpler.

Reply:
I believe It should be 
```
elif (settings.FEATURES.get('COURSE_DEFAULT_INVITE_ONLY') or invitation_only) and not can_enroll:
  ...
```